### DISCUSSION TITLE
How do I check if TOR is being used correctly by dnscrypto-proxy ?

### DISCUSSION THREAD

**Discussion by kuolemaaa** [DNSCrypt/dnscrypt-proxy#2694] [Q&A]
A part from checking in the config file that `proxy = 'socks5://127.0.0.1:9050'` and `force_tcp: true`

**Comment by lifenjoiner** [comment]
Similar to https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Checking.
* Stop the proxy/TOR.
* Use traffic capture.
